src/utils/errorHelper.ts

```diff
@@ -0,0 +1,10 @@
+export function checkError(response: Response) {
+  if (!response.ok) {
+    throw Error(response.statusText);
+  }
+  return response;
+}
+
+export function handleError(error: Error) {
```

Comment:
How exactly do you plan to handle some specific (let's say forbidden or unauthorized) errors using this?

Reply:
I'll move the error handling logic